Code2Gether-Discord-Bot.Library/BusinessLogic/PingLogic.cs

```diff
@@ -43,7 +43,7 @@ public Embed Execute()
                     .WithAuthor(_context.Message.Author)
                     .Build();
 
-                temporaryMessage.DeleteAsync();
+                temporaryMessage.DeleteAsync().Wait();
```

Comment:
This isn't quite what I had in mind. We need to make use of the `async` keyword in C#.